src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md

```diff
@@ -23,58 +24,69 @@ to show how real users experience the page and the origin in
-- **FID** measures the interactivity of the page, from the user's first
-  interaction to the time the page responds to it.
 - **LCP** measures the time until the page displays what is likely its main
   content, like a hero image or heading.
+- **FID** measures the interactivity of the page, from the user's first
+  interaction to the time the page responds to it.
 - **CLS** measures the degree of layout instability on the page, due to shifts
   like asynchronously loaded content being injected.
 
+### Additional metrics
+
+- **FCP** measures the time until the page displays something in the
+ foreground, like some text or a logo.
+- **INP** is an experimental metric that measures the latency of all interactions a user has made with the page, and reports a single value which all (or nearly all) interactions were below.
+- **TTFB** measures the the time between the request for a resource and when the first byte of a response begins to arrive.
+
+### Metric thresholds
+
+This table describes how values for these metrics are categorized as either
+"good", "needing improvement", or "poor".
+
 Metric | "Good" | "Needs Improvement" | "Poor"
 -- | -- | -- | --
-FCP | 0–1000ms | 1000ms–3000ms | 3000ms+
-FID | 0–100ms | 100–300ms | 300ms+
 LCP | 0–2500ms | 2500–4000ms | 4000ms+
+FID | 0–100ms | 100–300ms | 300ms+
 CLS | 0.00-0.10 | 0.10–0.25 | 0.25+
+FCP | 0–1800ms | 1800ms–3000ms | 3000ms+
+INP | 0–200ms | 200ms–500ms | 500ms+
+TTFB | 0–800ms | 800ms–1800ms | 1800ms+
 
-This table describes how values for these metrics are categorized as either
-"good", "needing improvement", or "poor".
+The three CWV metrics are the most important metrics, and are directly experienced by the users. The other metrics can influence those metrics but may not directly equate to user experience so it's not absolutely necessary that sites meet the "good" threshold, provided that it doesn't impede their ability to score well on the metrics that matter.
 
 There are three ways that the user experience is displayed in PSI:
 
 - a label summarizing the page as passing or not passing the Core Web Vitals
   assessment
 - percentiles measured in seconds or milliseconds (CLS is unitless)
-- a distribution representing the percent of "good", "needing improvement", and
-  "poor" experiences
+- a distribution representing the percent of "good", "needing improvement", and "poor" experiences is available by clicking on "Expand View" in the top right of this section.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/6ykCy8x82DN4syjcxoyJ.png", alt="Expanded view of CrUX data in PageSpeed Insights", width="800", height="719" %}
 
 In the screenshot above, the page is labelled as "passing" the Core Web Vitals
-assessment. To pass, the percentile must be categorized as "good".
+assessment. To pass, the percentile must be categorized as "good" in all three
+Core Web Vitals (FID can be optional for pages that do not have interactions).
 Otherwise, the page is labelled as "not passing".
```

Comment:
```suggestion
assessment. To pass, the percentile must be categorized as "good" in all three
Core Web Vitals. Otherwise, the assessment appears as "failed".  Some pages may not have sufficient FID data, in which case the page is assessed on the other two Core Web Vitals metrics.
```